pages/start_page.py

```diff
@@ -0,0 +1,63 @@
+from time import sleep
+
+from constants.start_page import StartPageConstants
+from pages.base_page import BasePage
+
+
+class StartPage(BasePage):
+
+    def __init__(self, driver):
+        super().__init__(driver)
+        self.constants = StartPageConstants()
+
+    def sign_up(self, username, email, password):
+        """Sign up as the user"""
+        # Fill Sign Up
+        self.fill_field(xpath=self.constants.SIGN_UP_USERNAME_FIELD_XPATH, value=username)
+        self.fill_field(xpath=self.constants.SIGN_UP_EMAIL_FIELD_XPATH, value=email)
+        self.fill_field(xpath=self.constants.SIGN_UP_PASSWORD_FIELD_XPATH, value=password)
+
+        sleep(1)
+        # Click button
+        self.click(xpath=self.constants.SIGN_UP_BUTTON_XPATH)
+        sleep(1)
+
+    def error_name(self, username):
+        """Sign Up with invalid name"""
+        self.fill_field(xpath=self.constants.SIGN_UP_USERNAME_FIELD_XPATH, value=username)
+
+    def error_password(self, password):
+        """Sign Up with invalid password"""
+        self.fill_field(xpath=self.constants.SIGN_UP_PASSWORD_FIELD_XPATH, value=password)
+
+    def error_email(self, email):
+        """Sign up twice with the same email"""
+        self.fill_field(xpath=self.constants.SIGN_UP_EMAIL_FIELD_XPATH, value=email)
+
```

Comment:
These methods are useless. Please reuse sign_up method with empty fields for such purpose